packages/compiler/test/decorator-utils.test.ts

```diff
@@ -154,8 +154,8 @@ describe("compiler: decorator utils", () => {
 
     it("emit diagnostics if using the same decorator on the same node", async () => {
       const diagnostics = await runner.diagnose(`
-        @tag
-        @tag
+        @global.tag
```

Comment:
Wondering if we are okay with making these changes to the tests, or if they are indicative that something is wrong about the implementation?

Reply:
is it because `global.tag` `TypeSpec.tag` are ambiguous?

Reply:
Yes, that's right. This happens quite a bit in tests where we just declare things on the global namespace.

Reply:
If that's the case I would maybe rename the test decorator to some other thing instead as this is kinda adding confusion to the test